### ISSUE TITLE
Crucial MX300/MX500 SATA SSDs show as locked, password unknown, on different machines

### ISSUE THREAD

**Issue by vogelfreiheit** [Drive-Trust-Alliance/sedutil#212]
I have tested this on a Lenovo laptop and an AMD Ryzen custom build workstation... in both cases the drives display as follows:

```
/dev/sdc ATA CT500MX500SSD1                           xxx xxx        
TPer function (0x0001)
    ACKNAK = N, ASYNC = N. BufferManagement = N, comIDManagement  = N, Streaming = Y, SYNC = Y
Locking function (0x0002)
    Locked = N, LockingEnabled = Y, LockingSupported = Y, MBRDone = N, MBREnabled = N, MediaEncrypt = Y
Geometry function (0x0003)
```

Both systems are dual boot with Windows 10 on a separate drive. These are data drives. I'm not quite sure what is happening, but obviously something initialized OPAL on the drives with admin1 passwords I don't have access to.

Anyone else?

**Comment by Vimalapril** [comment]
Can you confirm if you see the below error?
"One or more header fields have 0 length.
Unable to take ownership" 

Does other SATA SSDs work?

**Comment by Vimalapril** [comment]
Can you do a PSID revert and check if Win10 preactivation disabled solve the issue?

**Comment by r0m30** [comment]
Win10 (and all versions of windows that support bitlocker) take ownership of OPAL drives during installation unless you stop it at the very beginning of the install by setting a registry key.

**Comment by oom-is** [comment]
In case anyone comes back to this, this "Windows takes ownership of SEDs for BitLocker/eDrive" feature appears to only affect full UEFI+GPT in my testing - so if you're still installing with MBR/Legacy boot you won't hit this.

More importantly, info on the registry key follows if you're doing a new Windows install with full UEFI.  I have seen mixed results with trying to do a PSID revert after finishing the Windows install - the PSID revert always works, and on one system it allowed me to use SEDutil to take ownership without data loss while on another it wiped the drive.  _The only other case I've found that may block the "take ownership" behavior is the Samsung vendor-specific feature that doesn't allow taking ownership of an SED until after a PSID revert, as noted in other issue threads._  

Source of the text below:  https://answers.microsoft.com/en-us/windows/forum/windows8_1-hardware/sed-edrive-unprovisioning/87148920-e749-4e8a-b674-d8d8d551981d

[S]et a registry entry prior to letting Windows Setup partition the drive:

[HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Windows\EnhancedStorageDevices]
"TCGSecurityActivationDisabled"=dword:00000001

In other words, when you boot the Win8 (or later) setup DVD, press shift-F10 to load a command prompt, then set this registry entry BEFORE continuing with Windows setup.
